src/App.js

```diff
@@ -1,16 +1,23 @@
+// import { render } from '@testing-library/react';
 import './App.css';
+import React, {useState} from 'react';
+
+// const multiplier = 0;
+// const chickenRate = 0;
+// const total = 0; 
+
+var start = Date.now()
```

Comment:
Why are you using the rather outdated `var` keyword as opposed to `let` or `const`?

Reply:
Because it works.

Reply:
This isn't being used though?

Reply:
It will be in the next branch

Reply:
But it isn't now, I just think we need to keep the history clean so this PR makes sense.

Reply:
I've already moved on from this branch I can't make the changes. You took so long on a call so I just had to move on

Reply:
ive resolved them.